.travis.yml

```diff
@@ -52,13 +52,13 @@ matrix:
     bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
   - env:
       CHEFSTYLE: 1
-    rvm: 2.1
+    rvm: 2.2.5
```

Comment:
Because they were all originally set to 2.2 (I don't know where the 2.1 is coming from), so I'm just adding a patch version.